README.md

```diff
@@ -536,7 +536,19 @@ Return the time and date that the response was sent.
 
 ### `Fieldhand::NetworkError`
 
-An error (descended from `StandardError`) to represent any network issues encountered during interaction with the repository. Any underlying exception is exposed in Ruby 2.1 onwards through [`Exception#cause`](https://ruby-doc.org/core-2.1.0/Exception.html#method-i-cause).
+An error (descended from `StandardError`) to represent any network issues encountered during interaction with the repository.  
+If the HTTP request is not successful (returning a status code other than 200), 
+a `NetworkError` exception will be raised containing the error message and the response object.  
+Any underlying exception is exposed in Ruby 2.1 onwards through [`Exception#cause`](https://ruby-doc.org/core-2.1.0/Exception.html#method-i-cause).
+
+#### Fieldhand::NetworkError.new(message[, response])
```

Comment:
Good point. I don't think the client would never have to initialise the error but I'd like to document what that error might look like. (i.e. the fact that `response` is optional). I more or less said that in my description but I wanted to add a code example. Can you think of a better alternative or should I just take it out? thanks